guides/release/models/index.md

```diff
@@ -106,22 +106,27 @@ lists the drafts for the currently logged in user.
 You might be tempted to make the component responsible for fetching that
 data and storing it:
 
-```javascript {data-filename=app/components/list-of-drafts.js}
-import Component from '@ember/component';
+```javascript {data-file=src/ui/components/list-of-drafts/component.js}
```

Comment:
MU was removed from octane (see 
https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html for a more detailed writeup), can you swap the filepath back?

```suggestion
```javascript {data-file=app/components/list-of-drafts.js}
```